src/sequences/OEISSequenceTemplate.ts

```diff
@@ -19,37 +15,38 @@ export default class OEISSequenceTemplate extends SequenceCached {
     name = 'OEIS Sequence Template'
     description = 'Factory for obtaining sequences from the OEIS'
     oeisSeq = true
-    modulo = 0n
     cacheBlock = 1000
-
-    params: SequenceParamsSchema[] = [
-        new SequenceParamsSchema('oeisId', 'text', 'OEIS ID', true, false),
-        new SequenceParamsSchema('name', 'text', 'Name', false, false),
-        new SequenceParamsSchema(
-            'numElements',
-            'number',
-            'Number of Elements (leave blank to fetch up to first 1000)',
-            false,
-            false
-        ),
-        new SequenceParamsSchema(
-            'modulo',
-            'number',
-            'Modulo to apply to the sequence',
-            false,
-            false
-        ),
-    ]
+    oeisId = ''
+    givenName = ''
+    modulo = 0n
+    params = {
+        oeisId: {value: '', displayName: 'OEIS ID', required: true},
+        givenName: {value: '', displayName: 'Name', required: false},
+        cacheBlock: {
+            value: this.cacheBlock,
+            displayName: 'Number of Elements',
+            required: false,
+            description:
+                'How many elements to try to fetch from the database.',
+        },
+        modulo: {
+            value: this.modulo,
+            displayName: 'Modulo',
+            required: false,
+            description:
+                'If nonzero, take the residue of each element to this modulus.',
+        },
+    }
 
     constructor(sequenceID: number) {
         super(sequenceID) // Don't know the index range yet, will fill in later
     }
 
     async fillCache(): Promise<void> {
-        const resp = await axios.get(
-            `http://${process.env.VUE_APP_API_URL}/api/`
-                + `get_oeis_values/${this.settings.oeisId}/${this.cacheBlock}`
-        )
+        const backendUrl
+            = `http://${process.env.VUE_APP_API_URL}/api/`
+            + `get_oeis_values/${this.oeisId}/${this.cacheBlock}`
+        const resp = await axios.get(backendUrl)
```

Comment:
I don't care whether we use `resp` or `res` or something else, but it should be consistent across the code base. I think whatever abbreviation we want to use should be documented somehow.

Reply:
Right now, this is the only response variable in frontscope, so there isn't an inconsistency issue.